msteams-platform/m365-apps/actions-in-m365-quick-start-guide.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Actions in Microsoft 365 quick start guide
```

Comment:
The instruction you listed here is more like how to run the sample code. I wonder if you want be more clear about where is this sample code, and this instruction a doc to run sample code?

Reply:
+1 I agree. How about we publish these instructions as readme with the sample? I'm not convinced we need to publish them in docs as well